test/unit/processing/siti-agri/transform-header.test.js

```diff
@@ -144,6 +145,13 @@ describe('Transform header', () => {
     expect(result.value).toBe(undefined)
   })
 
+  test('for CS return Payment Type', async () => {
+    const filename = 'SITICS0001_AP_20230315084313836.dat'
+    const headerData = ['H', 'CS000000001', '001', 'A0000001', '1', '1000000001', 'GBP', 'abc', 'NE00', 'GBP']
+    const result = transformHeader(headerData, cs.schemeId, filename)
+    expect(result.paymentType).toBe('1')
```

Comment:
Suggest parsing this as an integer rather than string as if it exists, it will always be a number.

Thoughts?

Reply:
Agree, will update.